proto/process/agent.proto

```diff
@@ -784,10 +784,12 @@ message Cluster {
 	uint64 memoryCapacity = 7;
 	uint64 cpuAllocatable = 8;
 	uint64 cpuCapacity = 9;
-	string resourceVersion = 10;
-	int64 creationTimestamp = 11;
-	repeated string tags = 12;
-	ResourceMetrics metrics = 13;
+	map<string, int32> extendedResourcesAllocatable = 10;
```

Comment:
You can't use the same id for fields that already has been released. You need to start with id 14. If you want to delete fields, keep them but please add a comment to say that they are unused (or use [deprecated = true];)

Reply:
Makes sense, thanks you. I fixed it.